src/amo/pages/Addon/index.js

```diff
@@ -437,6 +437,7 @@ export class AddonBase extends React.Component {
 
     return (
       <Page
+        addonType={addon ? addonType : undefined}
```

Comment:
Instead of passing `undefined`, we could lookup the addon id anywhere in the store (but mostly in search results or landing data) and hopefully retrieve an add-on type.

Reply:
This is a nice idea in theory, but please see my comment at https://github.com/mozilla/addons-frontend/issues/10773#issuecomment-881518731.